fmod/src/steamaudio_fmod.cpp

```diff
@@ -512,3 +513,8 @@ void F_CALL iplFMODRemoveSource(IPLint32 handle)
 
     gSourceManager->removeSource(handle);
 }
+
+void F_CALL iplFMODSetGlobalHRTFDisabled(bool disabled)
```

Comment:
That would make it less clear that disabling HRTF using this function has global scope. Since there's no source parameter, the requested change wouldn't be too confusing, though.

Reply:
I see now that the suggested name is more consistent with other function names. Done via a6b8dab59669305b73ece9c55d2336e25849a4b9